.github/workflows/push.yml

```diff
@@ -10,7 +10,17 @@ jobs:
     steps:
       - name: Check out the repo
         uses: actions/checkout@v4
-      - name: Build image and run tests
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ env.DOCKERHUB_USER }}
+          password: ${{ env.DOCKERHUB_TOKEN }}
+
+      - name: Build and run tests for PHP 8.2
+        run: |
+          docker compose run ci
+
+      - name: Build and run tests for PHP 7.4
         run: |
-          docker login --username "$DOCKERHUB_USER" --password "$DOCKERHUB_TOKEN"
           docker compose run ci
```

Comment:
Proc to nejsou dva separatni joby, aby to jelo paralelne? Co rovnou pouzit matrix?